setup-tiger-eon.sh

```diff
@@ -236,17 +234,18 @@ create_tiger_db() {
 
     log_success "Tiger database created with service ID: $TIGER_SERVICE_ID"
 
-    # Get connection string
-    log_info "Retrieving database connection string..."
-
-    dsn=$(${tigerCmd} db connection-string --with-password ${TIGER_SERVICE_ID} 2>&1)
+    dsn=$(${jqCmd} -r '.connection_string // empty' <<< "$createResponse")
+    if [[ -z "$dsn" ]]; then
+        log_error "Failed to get connection string from response"
+        return 1
+    fi
 
     # Parse DSN: postgresql://user:password@host:port/database?params
     PGUSER=$(echo "$dsn" | sed -n 's|postgresql://\([^:]*\):.*|\1|p')
-    PGPASSWORD=$(echo "$dsn" | sed -n 's|postgresql://[^:]*:\([^@]*\)@.*|\1|p')
-    PGHOST=$(echo "$dsn" | sed -n 's|postgresql://[^@]*@\([^:]*\):.*|\1|p')
-    PGPORT=$(echo "$dsn" | sed -n 's|postgresql://[^@]*@[^:]*:\([^/]*\)/.*|\1|p')
     PGDATABASE=$(echo "$dsn" | sed -n 's|postgresql://[^/]*/\([^?]*\).*|\1|p')
```

Comment:
I guess we should add `user` and `database` to the `tiger service create` output. For now, we can keep parsing the connection string.

Reply:
I think for consistency would be good to output the info from the CLI still. 

Fine to hardcode or just leave the parse for right now though.

Reply:
Yeah, I really want to get it from the CLI. But agreed that there's no risk of it having a different value (for now)